lib/truffle/java/java_utilities.rb

```diff
@@ -271,7 +272,8 @@ def self.make_proxy(a_class)
             Interop.from_java_string(
               Java.invoke_java_method(CLASS_GET_SIMPLE_NAME, a_class)),
             a_proxy) unless parent == nil
-        rescue # rubocop:disable Lint/HandleExceptions
+        rescue
```

Comment:
Should this only rescue NameError and only the `const_set` be in the begin/rescue?